src/model/user.rs

```diff
@@ -192,7 +192,9 @@ impl CurrentUser {
             let mut pagination = http
                 .as_ref()
                 .get_guilds(
-                    &GuildPagination::After(guilds.last().map_or(GuildId(1), |g: &GuildInfo| g.id)),
+                    Some(&GuildPagination::After(
```

Comment:
I think this should be changed to `None`, as it's effectively trying to workaround this very issue.

Reply:
Updated